server/models/Update/add_ohada_balance_sheet.sql

```diff
@@ -0,0 +1,8 @@
+/* Add report in the navigation tree */
+INSERT INTO unit VALUES 
+  (206, '[OHADA] Bilan','TREE.OHADA_BALANCE_SHEET','',144,'/modules/reports/ohada_balance_sheet_report','/reports/ohada_balance_sheet_report');
```

Comment:
Can you remove the `id` here?

I can imagine that other developers will put in there own units with conflicting IDs.  Since we aren't sure which will get merged first, it makes sense to let the database assign the IDs if possible.

Reply:
The `id` column is not auto increment in the unit table

Reply:
Oh ... I suppose that makes sense.